<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtFirstName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtLastname"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Dialouge</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textDialog"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnClear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSubmit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
